usecases/optimization_paper/dodo.py

```diff
@@ -0,0 +1,65 @@
+import pathlib
+from workflow_graph.paper_workflow_graph import paper_workflow_graph
```

Comment:
What is that? (I stumbled upon this in the first task). Is that the diagramm of the workflow automatically being generated?

Reply:
YEs